README.md

```diff
@@ -1,2 +1,85 @@
-# webpack
-This is a webpack boilerplate
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-Do List
+
+> In this project, I built a simple HTML list of To Do tasks. The list is styled according to the specifications listed in the lesson. This simple web page is built using webpack and served by a webpack dev server.
+
+## Built With
+
+  *Major languages:*
+
+- HTML
+- CSS
+- JavaScript
+- Webpack
+- Webpack dev server
+
+## Live Demo
+
+[To-Do list structure](https://krisbobo.github.io/awesomebooks-ES6/)
+
+## Getting Started
+
+To get a local copy up and running, follow these simple example steps.
+
+## Prerequisites
+
+Before attempting to get a copy of this, you should know(have) how to do the following:
+
+- How to use the command line interface(CLI)
+
+- Know how to navigate directories or folders at the CLI.
+
+- Know how to get the URL(https/ssh) of a repository on GitHub.
+
+- You should have a code editor installed, preferably VsCode.
+
+## Setup
+
+- In desired folder or directory in the CLI, run the command:
+
+>                   git clone git@github.com:krisbobo/todo-list.git
+
+-Navigate into the cloned folder or repository by running the command:
+
+>                   cd <folder/name of the repository>
+
+- If VsCode is your default code editor, run:
+
+>                   code .
+
+- You are all setup!
+
+## Usage
+
+- While in the desired directory, run the code:
+
+>                   npm start
+
+## Authors
+
+👤 **Kiyong Bohbo**
+
+- GitHub: [krisbobo](https://github.com/krisbobo)
+- Twitter: [ChrisboboSys](https://twitter.com/ChrisboboSys)
+- LinkedIn: [kiyong-christian](https://linkedin.com/in/kiyong-christian)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [OPTIONAL] The MIT link is broken. Try to fix it. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.